packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,27 +1,50 @@
-import { CreateTaskCommandDTO } from 'shared'
+import { ObjectId } from 'mongoose'
+import { CreateTaskCommandDTO, UpdateTaskCommandDTO } from 'shared'
 import DashboardModel from './Dashboard'
 import { Task } from '../task/task'
 
 export class DashboardRepository {
   private readonly _dashboardModel = DashboardModel
 
+  // findIndexInDatabase is looking for index by id in an Array of Objects
```

Comment:
Code should be self explanatory, and comments explaining what the function does are not needed.

Reply:
Deleted